quill-core/src/test/scala/io/getquill/norm/FlattenOptionOperationSpec.scala

```diff
@@ -24,7 +25,31 @@ class FlattenOptionOperationSpec extends Spec { // hello
         o.getOrElse(1)
       }
       new FlattenOptionOperation(AnsiConcat, TraceConfig.Empty)(q.ast.body: Ast) mustEqual
-        If(BinaryOperation(Ident("o"), EqualityOperator.`_!=`, NullValue), Ident("o"), Constant.auto(1))
+        IfExist(o, o, c1)
```

Comment:
Why are we changing the test here? 🤔

Reply:
It's just a simplification, they are equivalent :)